### ISSUE TITLE
help！

### ISSUE THREAD

**Issue by cuiyijun12** [clovaai/stargan-v2#91]
在lpips.py中

# calculate the average of pairwise distances among all random outputs
    for i in range(num_rand_outputs-1):
        for j in range(i+1, num_rand_outputs):
            lpips_values.append(lpips(group_of_images[i], group_of_images[j]))
    lpips_value = torch.mean(torch.stack(lpips_values, dim=0))
出现错误

RuntimeError
stack expects a non-empty TensorList
  File "D:\biyelunwen\star\stargan-v2\metrics\lpips.py", line 97, in calculate_lpips_given_images
    lpips_value = torch.mean(torch.stack(lpips_values, dim=0))
  File "D:\biyelunwen\star\stargan-v2\metrics\eval.py", line 95, in calculate_metrics
    lpips_value = calculate_lpips_given_images(group_of_images)
  File "D:\biyelunwen\star\stargan-v2\core\solver.py", line 170, in train
    calculate_metrics(nets_ema, args, i+1, mode='latent')
  File "D:\biyelunwen\star\stargan-v2\main.py", line 49, in main
    solver.train(loaders)
  File "D:\biyelunwen\star\stargan-v2\main.py", line 172, in <module>
    main(args)

**Comment by 1xiaoliang** [comment]
I also have this question, did you have sovled this question?

**Comment by 1xiaoliang** [comment]
you can check you parameter --num_outs_per_domain, this parameter must >1

**Comment by 1xiaoliang** [comment]
You can check your parameter --num_outs_per_domain, this parameter must >1
